scripts/extractStrings.js

```diff
@@ -278,8 +278,8 @@ function generateLocProjectEntry(templatePath, resjsonOutputPath, rootDirectory)
 function getLocOutputPath(templatePath, extensionType, root) {
   const newTemplatePath = templatePath.replace(root, root.concat(LangOutputSpecifier))
   const templateSplit = newTemplatePath.split("\\");
```

Comment:
interesting, your stuff uses `\\` windows style paths?  the stuff in the tests using node is using `/` style unix paths? they both work locally and i the build infra?